### PR TITLE
Add Acquire Lease API

### PR DESCRIPTION
**Pull request by erezbo**

### CODE REVIEW

File: dome9ApiV2Py/dome9ApiV2Py.py [hunk 1]
```diff
@@ -266,6 +266,26 @@ def cloudAccountSyncNow(self, ID, outAsJson=False):
 			print(json.dumps(apiCall))
 		return apiCall
 
+	def acquireAwsLease(self, cloudAccountId, securityGroupId, ip, portFrom, portTo=None, protocol='ALL', length='1:0:0', region=None, accountId=None, name=None, outAsJson=False):
```
Comment (paveli-d9) [inline dome9ApiV2Py/dome9ApiV2Py.py]: Use none

File: dome9ApiV2Py/dome9ApiV2Py.py [hunk 2]
```diff
@@ -266,6 +266,26 @@ def cloudAccountSyncNow(self, ID, outAsJson=False):
...
+			'ip'             : ip,
+			'portFrom'       : portFrom,
+			'portTo'         : portTo,
+			'protocol'       : protocol,
+			'length'         : length,
+			'region'         : region,
+			'accountId'      : accountId,
+			'name'           : name
+		}
+		data = {key: temp_data[key] for key in temp_data if temp_data[key] is not None}
```
Comment (paveli-d9) [inline dome9ApiV2Py/dome9ApiV2Py.py]: data = {key: value for key, value in temp_data if value is not None}

**Review by paveli-d9 — CHANGES_REQUESTED** [review]

**Review by uditamar — CHANGES_REQUESTED** [review]
@erezbo please squash

**Review by paveli-d9 — COMMENTED** [review]
@udidome9 LGTM, are you done with the review?

**Review by uditamar — CHANGES_REQUESTED** [review]
please squash

**Review by erezweinstein5-zz — APPROVED** [review]

**Review by paveli-d9 — CHANGES_REQUESTED** [review]
Please rewrite commit message:
1. Capitalize the headline.
2. Use imperative language.
3. Clean lines which describe temporary minor fixes
4. Write a short documentation for new API.

**Review by paveli-d9 — APPROVED** [review]

**Pull request merged**